examples/js_binary/BUILD.bazel

```diff
@@ -457,21 +458,26 @@ js_binary(
     entry_point = "case10.js",
 )
 
-write_file(
+expand_template(
     name = "write10_launch_sh",
     out = "launch_case10.sh",
-    content = ["""\
-        # --- begin runfiles.bash initialization v2 ---
-        set -uo pipefail; f=bazel_tools/tools/bash/runfiles/runfiles.bash
-        source "${RUNFILES_DIR:-/dev/null}/$f" 2>/dev/null || \
-        source "$(grep -sm1 "^$f " "${RUNFILES_MANIFEST_FILE:-/dev/null}" | cut -f2- -d' ')" 2>/dev/null || \
-        source "$0.runfiles/$f" 2>/dev/null || \
-        source "$(grep -sm1 "^$f " "$0.runfiles_manifest" | cut -f2- -d' ')" 2>/dev/null || \
-        source "$(grep -sm1 "^$f " "$0.exe.runfiles_manifest" | cut -f2- -d' ')" 2>/dev/null || \
-        { echo>&2 "ERROR: cannot find $f"; exit 1; }; f=; set -e
-        # --- end runfiles.bash initialization v2 ---
-
-        $(rlocation aspect_rules_js/examples/js_binary/bin10-js_binary) "$@"
+    data = [":bin10-js_binary"],
+    is_executable = True,
+    substitutions = {
+        "{rlocationpath}": "$(rlocationpath :bin10-js_binary)",
+    },
+    template = ["""\
+# --- begin runfiles.bash initialization v2 ---
```

Comment:
nit: we have the bash runfiles lib in a starlark constant exposed by bazel-lib, you could probably load that instead of copy-paste

Reply:
good idea